### PR TITLE
Starting the translation of documentation for Portuguese

### PR DESCRIPTION
**Pull request by AnthonyMRodrigues**
…umentação

Thanks for submitting a PR! Please read these instructions carefully:

- [x] Explain the **motivation** for making this change.
- [ ] Provide a **test plan** demonstrating that the code is solid.
- [ ] Match the **code formatting** of the rest of the codebase.
- [x] Target the `master` branch, NOT a "stable" branch.

## Motivation (required)

What existing problem does the pull request solve?
This PR resolve the translation of documentation for portuguese.

## Test Plan (required)

A good test plan has the exact commands you ran and their output, provides screenshots or videos if the pull request changes UI or updates the website. See [What is a Test Plan?][1] to learn more.  

Don't need tests, only people who speak portuguese.

If you have added code that should be tested, add tests.

## Next Steps

Sign the [CLA][2], if you haven't already.

Small pull requests are much easier to review and more likely to get merged. Make sure the PR does only one thing, otherwise please split it.

Make sure all **tests pass** on both [Travis][3] and [Circle CI][4]. PRs that break tests are unlikely to be merged.

For more info, see the ["Pull Requests"][5] section of our "Contributing" guidelines.

[1]: https://medium.com/@martinkonicek/what-is-a-test-plan-8bfc840ec171#.y9lcuqqi9
[2]: https://code.facebook.com/cla
[3]: https://travis-ci.org/facebook/react-native
[4]: http://circleci.com/gh/facebook/react-native
[5]: https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#pull-requests

### CODE REVIEW

File: README-portuguese.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# React Native [![Build Status](https://travis-ci.org/facebook/react-native.svg?branch=master)](https://travis-ci.org/facebook/react-native) [![Circle CI](https://circleci.com/gh/facebook/react-native.svg?style=shield)](https://circleci.com/gh/facebook/react-native) [![npm version](https://badge.fury.io/js/react-native.svg)](https://badge.fury.io/js/react-native)
+
+React Native permite vocẽ a crie experiências de plataformas nativas usando uma consistente experiência de desenvolvimento baseado no JavaScript e [React](https://facebook.github.io/react). O foco do React Native é desenvolver de forma eficiente para a todas as plataformas que você deseja - aprenda uma vez, escreva em qualquer lugar. Facebook usa o React Native na produção de multiplos aplicativos e irá continuar investindo no React Native.
```
Comment (nfcampos) [inline README-portuguese.md]: "uma consistente experiência de desenvolvimento" -> "uma experiência de desenvolvimento consistente"

Comment (nfcampos) [inline README-portuguese.md]: "para a todas as plataformas" -> "para todas as plataformas"

Comment (AnthonyMRodrigues) [inline README-portuguese.md]: @nfcampos  Fixed on commit 95f3ec333bcdc0500bac81aa7f6bfa8fe1e179c4.
Thanks.

**Comment by AnthonyMRodrigues** [comment]
@hramos You are welcome.
Can I continue sending the documentation translated?
Because I wanna translate first, to help people who speak portuguese and do not speak english, then start with the code.

**Comment by hramos** [comment]
You could continue sending PRs, but I am worried that your effort will be lost once we add support for additional languages. For reference, this is how we are handling localization in the Jest website: https://crowdin.com/project/jest

You'll notice how Crowdin ingests our docs and then provides a portal for submitting translated strings. We use Crowdin as it integrates with our website generation system and takes care of providing all the localized markdown files for each language.

**Comment by AnthonyMRodrigues** [comment]
@hramos  I undertand, thanks for the answer.
I'll translate the documentation for another projects to help.
And send PR's for you only of changes of the code.

**Pull request closed without merging**